examples/knowledge_generation_using_nemotron/blocks/blocks.py

```
@@ -0,0 +1,62 @@
+
+
+@BlockRegistry.register("PostProcessThinkingBlock")
+class PostProcessThinkingBlock(Block):
+    def __init__(self, block_name: str, column_name: str) -> None:
+        super().__init__(block_name=block_name)  
+        self.column_name = column_name
+    
+    
+    def generate(self, samples: Dataset):
+        def post_process_thinking(x):
+            if '</think>' in x[self.column_name]:
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)